assets/js/data/cart/actions.js

```diff
@@ -181,3 +194,33 @@ export function* removeItemFromCart( cartItemKey ) {
 
 	yield itemQuantityPending( cartItemKey, false );
 }
+
+/**
+ * Changes the quantity for specified cart item:
+ * - Calls API to set quantity.
+ * - If successful, yields action to update store.
+ * - If error, yields action to store error.
+ * - Sets cart item as pending while API request is in progress.
+ *
+ * @param {string} cartItemKey Cart item being updated.
+ * @param {number} quantity Specified (new) quantity.
+ */
+export function* changeCartItemQuantity( cartItemKey, quantity ) {
+	yield itemQuantityPending( cartItemKey, true );
+
+	try {
+		const result = yield apiFetch( {
+			path: `/wc/store/cart/items/${ cartItemKey }&quantity=${ quantity }`,
```

Comment:
```suggestion
			path: `/wc/store/cart/items/${ cartItemKey }?quantity=${ quantity }`,
```
`cartItemKey` is not a query var, but part of the url, so `quantity` is the first var here, meaning you need to append it with `=` or else the API will return 404 route not found

Reply:
Fixed this now - e7b54d261adc71a9d8138be8a28ef18153a3ff8f

Interestingly the API request was working correctly even with the naughty url, I guess our server/WP is robust to this kind of abuse!